submissions/adeyemiJuwon/phpDatabase/customers.php

```diff
@@ -0,0 +1,40 @@
+<?php
+  include('cookies.php');
+  
+  if(!isset($_COOKIE['customers'])) {
+  echo "Cookie named '" .'customers' . "' is not set!";
+} else {
+  echo "Cookie '" . 'customers' . "' is set!<br>";
+  echo "Value is: " . $_COOKIE['customers'];
+}
+try{
+    
+    if($results ->rowCount() > 0){
```

Comment:
The `$results` that you are using here is that from the SQL query. It means you are not using that stored in `cookie` you are always hitting the database